### PR TITLE
CHANGELOG.md: Update changelog for 2.3 release

### PR DESCRIPTION
**Pull request by Holzhaus**
I checked with `git log 0e0c72ededdeb7520cd09b63731b4b05f8421db1..upstream/2.3 --merges`. Hopefully I didn't forget anything.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -65,27 +69,42 @@
...
+* Gracefully handle MIDI overflow [#825](https://github.com/mixxxdj/mixxx/pull/825)
+
+### Other ###
+* Add CMake build system with `ccache` and `sccache` support for faster compilation times and remove SCons [#2280](https://github.com/mixxxdj/mixxx/pull/2280) [#3618](https://github.com/mixxxdj/mixxx/pull/3618)
 * Make Mixxx compile even though `QT_NO_OPENGL` or `QT_OPENGL_ES_2` is defined (fixes build on Raspberry Pi) [lp:1863440](https://bugs.launchpad.net/mixxx/+bug/1863440) [#2504](https://github.com/mixxxdj/mixxx/pull/2504)
-
-### Packaging ###
-* Fix missing manual in deb package [lp:1889776](https://bugs.launchpad.net/mixxx/+bug/1889776)
+* Fix ARM build issues [#3278](https://github.com/mixxxdj/mixxx/pull/3602)[#3602]
+* Fix missing manual in DEB package [lp:1889776](https://bugs.launchpad.net/mixxx/+bug/1889776)
```
Comment (Swiftb0y) [inline CHANGELOG.md:103]:
```suggestion
* Fix missing manual in DEB package [lp:1889776](https://bugs.launchpad.net/mixxx/+bug/1889776)[#2985](https://github.com/mixxxdj/mixxx/pull/2985)
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,12 +36,16 @@
 * Workaround Linux skin change crash [#3144](https://github.com/mixxxdj/mixxx/pull/3144) [lp:1885009](https://bugs.launchpad.net/mixxx/+bug/1885009)
 * Fix touch control [lp:1895431](https://bugs.launchpad.net/mixxx/+bug/1895431)
 * Fix broken knob interaction on touchscreens [#3512](https://github.com/mixxxdj/mixxx/pull/3512)
+* AutoDJ: Make "enable" shortcut work after startup [#3242](https://github.com/mixxxdj/mixxx/pull/3242)
+* Add rate range indicator [#3693](https://github.com/mixxxdj/mixxx/pull/3278)[#3495]
```
[the remark this answers is not in the document]
Comment (Holzhaus) [reply CHANGELOG.md:40]: Oops, i'll investigate

**Review by Swiftb0y — APPROVED** [review]
LGTM, anything else to do?

**Comment by Holzhaus** [comment]
I hope not.

**Comment by Swiftb0y** [comment]
So Merge?

**Comment by Holzhaus** [comment]
Yup

**Comment by Swiftb0y** [comment]
I guess we can move this card to done? https://github.com/mixxxdj/mixxx/projects/2#card-60171536

**Comment by Holzhaus** [comment]
We also need the version history in the manual first :D

**Pull request merged**